### PR TITLE
fix: [CDS-41907]: Overflowing last published version label fixed

### PR DESCRIPTION
**Pull request by samarth-sharma31**
#### Screenshots

Before ->

Screenshot 2022-08-17 at 11 11 21 AM

After -->

Screenshot 2022-08-17 at 1 45 00 PM

Screenshot 2022-08-17 at 1 45 51 PM

https://user-images.githubusercontent.com/96409598/185071349-ca6830df-3539-4757-b143-6f60000387fe.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10619)

**Comment by samarth-sharma31** [comment]
retrigger titlecheck

File: src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx [hunk 1]
```diff
@@ -306,9 +306,17 @@ const BasicTemplateDetails = (
                             padding={'small'}
                             margin={{ bottom: 'medium' }}
                           >
-                            <Layout.Horizontal spacing="small" flex={{ justifyContent: 'start' }}>
+                            <Layout.Horizontal
+                              spacing="small"
+                              flex={{ justifyContent: 'start' }}
+                              style={{ maxWidth: '100%' }}
```
Comment (yc147) [inline src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx:312]: Avoid inline style.. just use `width="100%"`

File: src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx [hunk 2]
```diff
@@ -306,9 +306,17 @@ const BasicTemplateDetails = (
...
+                            >
                               <Icon name="info-messaging" size={18} />
-                              <Text color={Color.BLACK} font={{ weight: 'semi-bold', size: 'small' }}>
+                              <Text
+                                color={Color.BLACK}
+                                font={{ weight: 'semi-bold', size: 'small' }}
+                                style={{ whiteSpace: 'nowrap' }}
```
Comment (yc147) [inline src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx:318]: Avoid inline style.. use class

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx (hunk 1 above)
Comment (samarth-sharma31) [reply src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx]: done

File: src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx (hunk 2 above)
Comment (samarth-sharma31) [reply src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx]: done

**Comment by samarth-sharma31** [comment]
retrigger jest

**Review by yc147 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**